docs/unity/part-1.md

```diff
@@ -31,9 +36,9 @@ Open Unity and create a new project by selecting "New" from the Unity Hub or goi
 
 ![UnityHub-NewProject](/images/unity-tutorial/UnityHub-NewProject.JPG)
 
-For Project Name use `client`. For Project Location make sure that you use your `SpacetimeDBUnityTutorial` directory. This is the directory that we created in a previous step.
+**⚠️ Ensure `3D (URP)` is selected** to properly render the materials in the scene!
```

Comment:
can we keep the word **Important** in here?

also, is this related to the shader issue you mentioned, or is it separate? you said that wasn't URP related right?

Reply:
Sure, adding back **important** - unrelated (URP issues show all magenta)